applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.h

```diff
@@ -324,5 +324,11 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ConstitutiveLawUtilities
         const double YoungModulus,
         const double PoissonRatio);
 
+    /**
+    * It calculates the shear modulus "G"
+    * @param rValues: The CL parameters
+    */
+    static double CalculateShearModulus(ConstitutiveLaw::Parameters rValues);
```

Comment:
```suggestion
    static double CalculateShearModulus(const Properties& rProps);
```

no need to pass the CLaw params

Reply:
and please add a test